src/components/App.tsx

```diff
@@ -56,7 +56,7 @@ export default function App({
     initialViewState,
     dismissedViewState,
   });
-  const confirmed = !!window?.airgap?.getConsent?.()?.confirmed;
+  const { confirmed } = airgap.getConsent();
```

Comment:
is it safe to assume airgap is defined? in diff it was handling undefined case

Reply:
Yeah this is actually already provided above in the function and I didn't even notice